src/redux/books/BooksSlice.js

```diff
@@ -1,44 +1,94 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+// import { addBook, removeBook, getBook } from '../api';
```

Comment:
* Kindly remove all commented code from your code base as it makes your codebase messy and unprofessional 🙏